blank_react_project/package.json

```diff
@@ -3,14 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "assemblyscript-json": "github:nearprotocol/assemblyscript-json",
-    "bignum": "github:MaxGraey/bignum.wasm",
     "gh-pages": "^2.1.1",
     "gulp": "^4.0.2",
     "js-cookie": "^2.2.1",
-    "near-runtime-ts": "^0.5.1",
-    "near-shell": "^0.18.0",
-    "nearlib": "^0.19.1",
```

Comment:
why remove nearlib?

Reply:
It's a dependency of nearshell. Should we change that?

Reply:
It's a dependency of near-shell, but also this project presumably uses it directly. I think it's generally good idea to list all direct dependencies in `package.json` (vs e.g. something like `near-hello` which isn't used directly here).

Reply:
Ah then should we list it as a peer dependency that way it will give a warning if the version of nearlib is incompatible with near-shell's?

Reply:
@willemneal I thought near-shell would use it's own version of nearlib unless we mess up semver?